projects/admin/src/app/menu/menu.component.ts

```diff
@@ -131,6 +131,14 @@ export class MenuComponent implements OnInit {
             routerLink: '/records/persons',
             iconCssClass: 'fa fa-user'
           }]
+        }, {
+          name: this.translateService.instant('Acquisitions'),
+          iconCssClass: 'fa fa-university',
```

Comment:
Not sure it's the fittest icon, but I cannot find a better one. :thinking:

Reply:
maybe `cart-arrow-down` (https://fontawesome.com/icons/cart-arrow-down?style=solid) ?

Reply:
Why not. But never mind, it does not matter so much, as long as the text is clear with a stable position.